src/floris/simulation/wake_velocity/curl.py

```diff
@@ -70,7 +70,7 @@ class CurlVelocityDeficit(BaseClass):
 
     model_grid_resolution: Union[List[float], Vec3] = attr.ib(
         default=[250, 100, 75],
-        converter=convert_to_Vec3,
+        # converter=convert_to_Vec3,
```

Comment:
Ah I deleted that function from utilities and commented it here so that the curl model could be imported. However, this is not currently used so I was planning to address this when we reenable curl. Maybe the `convert_to_vec3` function should be moved to the curl model for future reference.